Components/button/button.css

```diff
@@ -0,0 +1,139 @@
+.VariantContainer
+{
+      margin-top: 25px;
+      border: 3px solid var(--primary-color);
+      width: 900px;
+      height: 200px;
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      gap: 25px;
+      justify-content: center;
+      align-items: center;  
+      text-align: center; 
+}
+  
+.btn
+{
+      width: 80px;
+      height: 45px;
+      padding: 8px;
+      border: none;
+      border-radius:5px;
+}
+  
+.btn a
+{
+      text-decoration: none;
+}
+  
+.btn i
+{
+      color: var(--white-color);
+}
+  
+.primary-btn
```

Comment:
follow a consistent pattern everywhere, for example :- use btn-primary, btn-secondary, btn-warning. You have followed this pattern in btn-success below, that is correct.